src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-alert */
+/* eslint-disable import/no-extraneous-dependencies */
+const Big = require('big.js');
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = new Big(numberOne);
+  const num2 = new Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      return num1.plus(num2).toString();
+    case '-':
+      return num1.minus(num2).toString();
+    case 'x':
+      return num1.times(num2).toString();
+    case '/':
+      return num1.div(num2).toString();
```

Comment:
Let's handle cases where num2's `numberTwo` is 0 (zero) since division by zero will throw an error.